.github/workflows/update-lts-tags.yaml

```diff
@@ -39,12 +39,12 @@ jobs:
         run: |
           CHANGES=$(git status --porcelain=v1 2>/dev/null | wc -l)
           if [ ${CHANGES} -ne 0 ]; then
-            gh pr list --state open | grep "Update appVersions for DC apps"
-            if [ $? -ne 0 ]; then
+            OPEN_PRS=$(gh pr list --state open | grep "Update appVersions for DC apps" || true)
+            if [[ ${OPEN_PRS} ]]; then
               echo "Not raising PR because there are open PRs to upgrade versions. Merge or close them to let the workflow raise new PRs"
             else
               git checkout -b lts-tag-update-${GITHUB_RUN_ID}
-              git add -A
+              git add src
```

Comment:
Would you please explain why we need this change?

Reply:
Must be leftovers from some experiments. This works too since the update script edits files in `src/main/charts` and `src/test`